### PR TITLE
Update helper text for report names

### PR DESCRIPTION
**Pull request by madelondohmen**
### Changes
This PR adds a different helper text to the report name section (Generate Report > Step 4: Export setup).
The old text didn't include instructions about the placeholders that have been added.

The text only changes when "Recurrence - Yes, repeat" is selected.

### Demo
afbeelding

### QA notes
- Generate a report
- Select "Yes, repeat" in the Recurrence section
- Text should be visible

---

### Code Checklist

- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] I have checked the documentation and made changes where necessary.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

- [ ] Tickets have been created for newly discovered issues.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by ammar92 — DISMISSED** [review]

File: rocky/reports/templates/partials/report_names_header.html [hunk 1]
```diff
@@ -3,9 +3,19 @@
...
-        to the name. To do so you can select a standard option or use a <a href="https://strftime.org/" target="_blank">Python
-        strftime code</a> in the report name.
-    {% endblocktranslate %}
+    {% if recurrence %}
+        {% blocktranslate trimmed %}
+            Define a custom name format for your report(s). This format will be applied to all
+            generated (sub)reports. To make the report names more descriptive,
+            you can include placeholders for the object name and/or the report type. Use the
+            placeholder "{ooi}" for the object name and "{report type}" for the report type.
+            For example, the format "{report type} for {ooi}" could generate: "DNS Report for example.com".
```
Comment (underdarknl) [inline rocky/reports/templates/partials/report_names_header.html:12]: We should note that users can also use the strftime placeholders to insert the Referencedate time into the report name at time of generation..

File: rocky/reports/templates/partials/report_names_header.html [hunk 2]
```diff
@@ -2,10 +2,33 @@
...
+        {% blocktranslate trimmed %}
+            Define a custom name format for your report(s). This format will be applied to all generated
+            (sub)reports.
+        {% endblocktranslate %}
+    </p>
+    <p>
+        {% blocktranslate trimmed %}
+            To make the report names more descriptive, you can include placeholders for the
+            object name, the report type and/or the reference date. Use the placeholder "{ooi}" for the
+            object name, "{report type}" for the report type and use a <a href="https://strftime.org/" target="_blank">Python
```
Comment (underdarknl) [inline rocky/reports/templates/partials/report_names_header.html:16]:
```suggestion
            object name, "{report type}" for the report type and use a <a href="https://strftime.org/" target="_blank" rel="noopener">Python
```

File: rocky/reports/templates/partials/report_names_header.html [hunk 3]
```diff
@@ -2,10 +2,33 @@
...
+        {% blocktranslate trimmed %}
+            For example, the format "{report type} for {ooi} at %x" could generate:
+            "DNS Report for example.com at 01/01/25".
+        {% endblocktranslate %}
+    </p>
+{% else %}
+    <p>
+        {% blocktranslate trimmed %}
+            Give your report(s) a custom name and optionally add the reports' reference date
+            to the name. To do so you can select a standard option or use a <a href="https://strftime.org/" target="_blank">Python
```
Comment (underdarknl) [inline rocky/reports/templates/partials/report_names_header.html:30]:
```suggestion
            to the name. To do so you can select a standard option or use a <a href="https://strftime.org/" target="_blank" rel="noopener">Python
```

**Comment by stephanie0x00** [comment]
## Checklist for QA:

- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
  - [x] I have managed to pass the onboarding flow
  - [x] Objects and Findings are created properly
  - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [x] I checked the logs for errors and/or warnings and made issues where necessary

### What works:
Explanation text really helps to tell the user how it works and what the placeholders do! 

### What doesn't work:
n/a

### Bug or feature?:
n/a

**Review by underdarknl — APPROVED** [review]

**Pull request merged**